.github/workflows/generatemd.yml

```diff
@@ -0,0 +1,66 @@
+name: Generate Proposal MD
+
+on:
+  workflow_dispatch: #periodic
+    inputs:
+      logLevel:
+        description: "Log lev"
+        required: true
+        default: "warning"
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  generateHTML:
+    env:
+      JBAL_VERSION: "2201.0.3"
```

Comment:
@bimsarapathiraja Do we need this as we use Ballerina GitHub Action?

Reply:
env part?

Reply:
removed